frontend/components/organization/Edit.jsx

```diff
@@ -257,6 +279,7 @@ Edit.propTypes = {
     fetchOrganization: PropTypes.func.isRequired,
     fetchRoles: PropTypes.func.isRequired,
     inviteToOrganization: PropTypes.func.isRequired,
+    resendInviteToOrganization: PropTypes.func.isRequired,
```

Comment:
This should be optional since not every user will get a resend invite.

Reply:
Thanks for catching that, @apburnes. Just pushed a fix.